flink-runtime/src/main/java/org/apache/flink/runtime/checkpoint/PendingCheckpoint.java

```diff
@@ -109,13 +109,13 @@ public boolean isDiscarded() {
 		return discarded;
 	}
 	
-	public CompletedCheckpoint toCompletedCheckpoint() {
+	public Checkpoint toCompletedCheckpoint() {
```

Comment:
Maybe we could rename this method to something `finalizeCheckpoint` or `completeCheckpoint` if it's not public.